manifests/d/DataTools/CATCLI/0.22.0.12/DataTools.CATCLI.locale.en-US.yaml

```diff
@@ -0,0 +1,60 @@
+Moniker: catcli
+Tags:
+- test
+- testing
+- data
+- data-testing
+- cli
+- cat
+Agreements:
+- Agreement: |
+    CAT SOFTWARE LICENSE
+    TERMS AND CONDITIONS
+
+    „Author” 	shall mean a company JOYFUL CRAFTSMEN s.r.o., having its registered office at V Přístavu 1585/10, 170 00 Prague 7, Czech Republic, ID No. 02937948, registered in the Commercial Register maintained by the Municipal Court in Prague, Section C, File 225495;
+    „CAT software“ 	shall mean a software product developed by the Author, an author's work, designed for data testing, whose features include speed, ease of test generation, ease of use and flexibility in data formats and sources designed for testing;
+    “License” 	shall mean terms and conditions for use of the CAT software as specified below;
+    “You”	shall mean an individual or a legal entity who downloads an appropriate version of the CAT software available at powershellgallery.com.
+
+    (i)	The Author hereby grants You an authorisation to exercise the right to use the CAT software free of charge. You are entitled to use the functionalities of the CAT software in interactive mode only, for your own needs only and in line with the purpose thereof, i.e. to test data on your server. Interactive mode shall mean that appropriate operation of CAT software is started manually by your own action.
+
+    (ii)	The CAT software is provided “as is”, without warranty of any kind, expressed or implied, including but not limited to any warranties of title, non-infringement, merchantability or fitness for a particular purpose. In no event shall be the Author liable for any claim, damages or other liability arising out from or in connection with the CAT software and its use.
+
+    (iii)	Without a prior written consent of the Author, You are not authorised to assign, convey, lend, allow to be used or otherwise provide on a temporary or permanent basis an authorisation comprising the Licence or the Licence to third parties or to make the CAT software available to the general public. The Licence does not include provision of source codes for the CAT Software. You are not authorised to carry out any activities leading to uncovering the ideas and principles behind the functioning of the CAT software in order to design an identically or similarly functioning software (the reverse engineering).
+
+    (iv)	Specific and more comprehensive works using the CAT software may be accessible under different terms upon registration.
+
+    (v)	By downloading the CAT software from powershellgallery.com You declare your understanding and acceptance of these terms and conditions of CAT software License.
+
+    (vi)	By downloading the CAT software You acknowledge that the Author may use dates related to operation of CAT software on your server for analytic and statistic purposes.
+
+    JOYFUL CRAFTSMEN s.r.o.
```

Comment:
```suggestion
```

Reply:
@Trenly I have a problem with this. I left `License: Commercial` and `LicenseUrl: https://docs.justcat.it/docs/license` in the manifest. But users are not prompted to agree the license text, which is required. If the license text cannot be in the manifest, how do I ensure users have to agree with the license text?
Is there anything we can do about the already published package? (we need to either hide it or require users to accept the license).
Thanks for help!

Reply:
@petrparik - That would require the `Agreements` field as you originally had it; However, that field is not currently accepted in the community repo. In order to use it you would need to set up your own public source that users could add.

cc @denelon re: verified publishers